includes/Experiments.php

```diff
@@ -354,7 +354,17 @@ public function get_experiments(): array {
 				'group'       => 'general',
 				'default'     => true,
 			],
-
+			/**
+			 * Author: @spacedmonkey
+			 * Issue: #8811
+			 * Creation date: 2021-09-06
+			 */
+			[
+				'name'        => 'disableArchive',
+				'label'       => __( 'Disable Archive', 'web-stories' ),
+				'description' => __( 'Disable custom post type archive for web stories.', 'web-stories' ),
+				'group'       => 'general',
```

Comment:
```suggestion
				'name'        => 'disableArchive',
				'label'       => __( 'Archive Page', 'web-stories' ),
				'description' => __( 'Allow disabling the Web Stories archive page' ),
				'group'       => 'general',
```

With #8812 in mind, should we make this more general?

```suggestion
				'name'        => 'storiesArchiveCustomization',
				'label'       => __( 'Archive Page', 'web-stories' ),
				'description' => __( 'Enable Web Stories archive page customization' ),
				'group'       => 'general',
```